.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules
-InlineHtmlStripStylesTransformer.js
+InlineFilesTransformer.js
+StripStylesTransformer.js
+TransformUtils.js
```

Comment:
I think it's time for a `build/` directory – ideally in a separate PR :)

Reply:
Will start with that once this is done!